### PR TITLE
Automated PR: Standardising Files

### PR DESCRIPTION
**Pull request by xorimabot**
This PR will standardise the files we have with out agreed spec in sous-chefs/repo-management, this repo has been identified as a cookbook repo by topic of chef-cookbook

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
+      "chef-software.chef",
+      "rebornix.ruby",
```
Comment (damacus) [inline .vscode/extensions.json:4]: I just got an error from VSCOde that it didn't exist! :(

Comment (JohnRoesler) [reply .vscode/extensions.json:4]: ruh roh. also newline @Xorima 😁

Comment (Xorima) [reply .vscode/extensions.json:4]: This one is all @tas50 I just put it in the right location

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**